doc/guide/running-software.md

```diff
@@ -35,22 +35,23 @@ tail -f uart0.log
 ## Running on the Sonata FPGA
 
 Any builds of software in this repository will also produce a UF2 file containing the built firmware.
-When the Sonata FPGA is plugged into the computer, it should show up as a drive called 'SONATA'.
+When the Sonata FPGA is plugged into the computer, it should show up as a drive called `SONATA`.
```

Comment:
I like to try and limit the over use of verbatim. In this case, I think it's better to use single quote as it is normally used: to differentiate the name. Then verbatim can be used for the absolute path.

Reply:
Ok, this should now be reverted.